Engine/src/Engine/ResourceManagement/ShaderGenerator.h

```diff
@@ -1,6 +1,13 @@
 #pragma once
 #ifndef SHADERGENERATOR_H
 #define SHADERGENERATOR_H
+
+#include "Engine/ResourceManagement/ResourceManager.h"
+#include <string>
+#include <map>
+
+using namespace std;
```

Comment:
guidelines: no using namespace in header files

Reply:
Done